File: Daily Challenge/Java/Matrix - Integers & Operators 31.8.2025.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int R = sc.nextInt();
        int C = sc.nextInt();
        String[][] mat = new String[R][C];

        // Input matrix
        for (int i = 0; i < R; i++) {
            for (int j = 0; j < C; j++) {
                mat[i][j] = sc.next();
            }
        }

        // Process matrix
        for (int i = 0; i < R; i++) {
            for (int j = 0; j < C; j++) {
                if (Character.isDigit(mat[i][j].charAt(0))) {
                    continue;
                }
                if (mat[i][j].equals("+")) {
                    int up = Integer.parseInt(mat[i - 1][j]);
                    int down = Integer.parseInt(mat[i + 1][j]);
                    mat[i][j] = String.valueOf(up + down);
                } else {
                    int up = Integer.parseInt(mat[i - 1][j]);
                    int down = Integer.parseInt(mat[i + 1][j]);
                    mat[i][j] = String.valueOf(Math.abs(up - down));
                }
            }
        }

        // Print result
        for (int i = 0; i < R; i++) {
            for (int j = 0; j < C; j++) {
                System.out.print(mat[i][j] + " ");
            }
            System.out.println();
        }

        sc.close();
    }
}

File: Daily Challenge/Python/Matrix - Integers & Operators 31.8.2025.py
R, C = map(int, input().split())
mat = [input().split() for _ in range(R)]

for i in range(R):
    for j in range(C):
        if mat[i][j].isdigit():
            continue
        elif mat[i][j] == '+':
            mat[i][j] = int(mat[i-1][j]) + int(mat[i+1][j])
        else:
            mat[i][j] = abs(int(mat[i-1][j]) - int(mat[i+1][j]))

for i in mat:
    print(*i)